src/routes/exercises/ExercisesScreen.tsx

```diff
@@ -90,9 +89,9 @@ const ExercisesScreen = ({ route, navigation }: ExercisesScreenProps): JSX.Eleme
           title={item.title}
           description={item.description}
           onPress={() => handleNavigation(item)}
-          arrowDisabled={nextExercise === null || item.level > nextExercise.level}>
-          <Trophy level={item.level} />
-        </ListItem>
+          arrowDisabled={nextExercise === null || item.level > nextExercise.level}
```

Comment:
Maybe it would be better to read Progress here and pass the results to the listItems?

Reply:
@SteffiStuffel just moved the feedback logic to `ExerciseScreen`. Good call of yours, this simplified the ListItem and FeedbackBadge. 

Sorry to request a re-review, but I'd be great if you could have a look at it again.